src/main/java/com/ericsson/ei/subscription/HttpRequestSender.java

```diff
@@ -94,7 +94,7 @@ public boolean postDataMultiValue(String notificationMeta,
                     notificationMeta, e.getMessage(), e);
             return false;
         } catch (Exception e) {
-            LOGGER.error(e.getMessage(), e);
+            LOGGER.error("HTTP-request failed when trying to connect to URL: {}", notificationMeta, e);
```

Comment:
Remove the dash in between HTTP and request.

Reply:
Done.